nservicebus/operations/performance-counters-criticaltime.include.md

```diff
@@ -0,0 +1,16 @@
+## Critical Time
+
+**Counter Name:** `Critical Time`
+
+**Added in:** Version 3
+
+Critical Time is the time from a message being sent until successfully processed. This metric is useful for monitoring message Service-level agreement. For example "All orders should be processed within X seconds/minutes/hours". Define an SLA for each endpoint and use the `CriticalTime` counter to ensure it is adhered to.
```

Comment:
> Define an SLA for each endpoint

how? can we clarify if it's a NSB setting, or just an internal system doco, or something else?